src/main/java/core/basesyntax/FileWork.java

```diff
@@ -1,14 +1,47 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Arrays;
 /**
  * <p>Дано файл, потрібно прочитати його вміст і вибрати всі слова що починаються на `w`.
  * Результат повернути у вигляді відсортованого масиву (за замовчуванням). Всі слова повинні
  * бути в нижньому регістрі. У випадку якщо таких слів не знайдено повернути пустий масив.
  * Приклад: Width world Wide web
  * Результат: web wide width world</p>
  */
+
 public class FileWork {
+    static final String LETTER = "w";
+    static final String POINT = ".";
+    static final String EMPTY = "";
+    static final String SPACE = " ";
+    static final String EXCLAMATION_MARK = "!";
+    static final String QUESTION_MARK = "?";
+
     public String[] readFromFile(String fileName) {
-        return null;
+        String tempString = EMPTY;
+        String[] result;
+        try (BufferedReader newReader = new BufferedReader(new FileReader(fileName))) {
+            String line;
+            while ((line = newReader.readLine()) != null) {
+                String[] arr1 = line.toLowerCase().split(SPACE);
+                for (String line2 : arr1) {
+                    if (line2.substring(0, 1).equals(LETTER)) {
```

Comment:
see if string have a special method to check whether the first character is equal to some

Reply:
done